src/v2/guide/components.md

```diff
@@ -229,6 +229,20 @@ If you try this in your template however, Vue will show an error, explaining tha
 </div>
 ```
 
+Here we pass entire post object to the `props` option instead of separately pass title and content:
```

Comment:
Adjusting the grammar, I think this reads better:

> Here, we pass the entire post object to the `props` option instead of separately passing the title and content.

Reply:
agree, aligned with your advice :)